src/background/index.ts

```diff
@@ -0,0 +1,36 @@
+declare const chrome: any;
+
+import { Filter, Details } from "./types";
+
+const urlRegex = /https:\/\/github\.com\/.+\/.+\/(issues|pull)\/\d+/;
```

Comment:
@JoshuaKGoldberg How do you feel about regex? 😆

Reply:
@JoshuaKGoldberg How do you feel about regex? 😆